programmers/카펫.js

```diff
@@ -0,0 +1,16 @@
+function solution(brown, yellow) {
+    const result = [];
+    let length  = 0;
+    
+    for (let i = 1; i < yellow + 1; i++) {
+        if (yellow % i === 0) {
+            length = parseInt(yellow / i);
```

Comment:
```suggestion
            length = Number(yellow / i);
```
사용하신 parseInt와 number는 둘 다 문자열을 Number타입으로 변환해주는데,
차이점으로는 
1. parseInt는 문자와 숫자가 섞여있어도 숫자만 걸러내어 주는 역할 
  ex ) 
let str = '2021년도';
console.log(parseInt(str));   // 2021
console.log(Number(str));  // Nan
다만, 시작이 문자일 경우 둘 다 NaN

2. parseInt는 소수점이 있을경우 소수점은 버리고, Number은 소수점까지 출력해준다.

Reply:
여기선 parseInt를 쓰는게 틀린건가요?

Reply:
두개 다 됩니다 ! 제가 말을 안했군요 ㅎㅎ